charts/tempo-distributed/values.yaml

```diff
@@ -1263,6 +1263,21 @@ config: |
     grpc_server_max_send_msg_size: {{ .Values.server.grpc_server_max_send_msg_size }}
     http_server_read_timeout: {{ .Values.server.http_server_read_timeout }}
     http_server_write_timeout: {{ .Values.server.http_server_write_timeout }}
+  {{- if .Values.memcached.enabled }}
+  cache:
+    caches:
+      - roles:
+          - parquet-footer
```

Comment:
missing `parquet-page` but this might be for the best. `parquet-page` is incredibly high volume and would forcibly flush out the other values.

Reply:
I saw the warning in the docs, that is why I didn't want to add it as the default.